packages/react-toolkit-monitor/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@osdk/react-toolkit-monitor",
+  "version": "0.0.1",
```

Comment:
Once this merges, this will publish. Is this what you want or do you want a beta package?